src/mds/MDBalancer.cc

```diff
@@ -402,6 +415,8 @@ void MDBalancer::handle_heartbeat(MHeartbeat *m)
                           << " : " << cpp_strerror(r);
       }
       prep_rebalance(m->get_beat());
+    
+      mds_load.clear();
```

Comment:
why clear mds_load clear. I think it's better to leave it for debug purpose

Reply:
To ensure mds doesn't take previous heartbeat info to the next rounds, especially for the mds other than mds0.

Reply:
Since there is already heartbeat check logic added in handle_heartbeat, I agree we could avoid the clear operation here in case we want to export this kind of mdsload info for debugging.